src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1125 @@
+  </Collapser>
+</CollapserGroup>
+
+## Infrastructure [#infrastructure]
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="host-monit]"
+    title="Host monitoring"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ✅ 
+
+    Host entities with golden metrics are available independent of the data source: New Relic infrastructure agent vs. OpenTelemetry host receiver.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="inventory"
+    title="Inventory"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+
+    The [infrastructure inventory](/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure) capability has no equivalent in OpenTelemetry, which is based on metrics, traces and logs.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="infra-events"
+    title="Infrastructure events"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+
+    Most infrastructure events depend on inventory or custom events pushed by the New Relic infrastructure agent.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="third-party-svc"
+    title="Third-party services"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+
+    [On-host integrations](/docs/infrastructure/infrastructure-integrations/get-started/introduction-infrastructure-integrations#on-host) (NGINX, MySQL, Redis, Apache, RabbitMQ, and many more) and the [Kubernetes integrations](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/) are required as a source for curated dashboards and entities in New Relic.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="k8s-integ"
+    title="Application/service integration with Kubernetes telemetry"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+
+    New Relic agent data for services shows up in the Kubernetes cluster explorer, but OpenTelemetry services data does not yet. We recommend that for any OpenTelemetry service using Kubernetes you should decorate your telemetry with the relevant resource attributes from the OpenTelemetry semantic conventions so you can query for this data.
+  </Collapser>
+</CollapserGroup>
+
+## Serverless [#serverless]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic:  🟡 Limited support
+
+OpenTelemetry serverless function telemetry can be reported, stored, and visualized with custom dashboards. A dedicated view of this telemetry is not yet available.
+
+## Alerts [#alerting]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 🟡 Limited support
+
+Since alerts on OpenTelemetry data are based on spans instead of metrics, alerting with OpenTelemetry has lower fidelity than alerting with New Relic agents.
```

Comment:
OTel supports alerts on metrics as well, so this isn't accurate. Let's remove the entire sentence.